src/test/resources/application-test.yml

```diff
@@ -1,9 +1,13 @@
 server:
   port: 0
 
-ribbon.http.client.enabled: true
+ribbon.http.client.enabled: false
```

Comment:
Do we need to update this on the service config too?
This value is set to true on the config server (in https://github.com/SMARTRACTECHNOLOGY/smartcosmos-cluster-config/blob/master/application.yml)